src/llmcompressor/modifiers/transform/spinquant/base.py

```diff
@@ -55,24 +55,34 @@ class SpinQuantModifier(Modifier, use_enum_values=True):
         A mapping will be inferred if None is provided
     :param norm_mappings: Specifies layers within a model to target for norm fusing.
         A mapping will be inferred if None is provided
-    :param transform_config: Optional transform config which overrides `mappings`
+    :param transform_config: Optional transform config for overriding provided arguments
     """
 
-    rotations: List[SpinquantRotation] = Field(default_factory=lambda: ["R1", "R2"])
+    rotations: List[SpinquantRotation] = Field(
+        default_factory=lambda: ["R1", "R2"], exclude=True
+    )
     transform_type: Literal["hadamard", "random-hadamard", "random-matrix"] = Field(
-        default="hadamard"
+        default="hadamard", exclude=True
     )
-    randomize: bool = Field(default=False)
-    learnable: bool = Field(default=False)
+    randomize: bool = Field(default=False, exclude=True)
+    learnable: bool = Field(default=False, exclude=True)
```

Comment:
@kylesayrs  why are we excluding these? wouldn't we want them to persist in json?

Reply:
Yep, let's keep them